README.rst

```diff
@@ -0,0 +1,220 @@
+# Herbie
+
+[![Build Status](https://travis-ci.org/project-a/herbie.svg?branch=master)](https://travis-ci.org/project-a/herbie)
```

Comment:
should we replace travis-ci with github ci?

Reply:
Yes totally :+1: